### DISCUSSION TITLE
windows 11 shutdown

### DISCUSSION THREAD

**Discussion by Nadim-os** [AlexGustafsson/homebridge-wol#185] [Questions]
any clue to shutdown on lan
I Tried all  command and never succeded thank you

**Comment by AlexGustafsson** [comment]
First, make sure to try the guide available here: https://github.com/AlexGustafsson/homebridge-wol/wiki/Running-on-Synology-Docker-and-turn-Windows-10-on-off-using-SSH-and-Samba.

If that doesn't work, I think your best bet is to ask this question over at https://superuser.com. You're looking for how to remotely turn your computer off using SSH. Any command that can be run in a terminal on your Homebridge server can be used with this plugin via the shutdown command.

If you end up solving the issue, please post your solution here as it helps others facing the same issue.

**Comment by duckdany** [comment]
1. Start remote registry service
(admin cmd:
sc config RemoteRegistry start= auto
sc start RemoteRegistry)
2. In registry set HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System\LocalAccountTokenFilterPolicy to 1 or add it.

After win 11 upgrade I had to set up these settings again.
